tests/networkpolicy_test.go

```diff
@@ -223,17 +320,44 @@ func assertPingFail(fromVmi, toVmi *v1.VirtualMachineInstance) {
 	}, 5*time.Second, 1*time.Second).ShouldNot(Succeed())
 }
 
-func waitForNetworkPolicyDeletion(policy *v1network.NetworkPolicy) {
+func createNetworkPolicy(policy *networkv1.NetworkPolicy) {
+	virtClient, err := kubecli.GetKubevirtClient()
+	ExpectWithOffset(1, err).ToNot(HaveOccurred())
+
+	By(fmt.Sprintf("Create %s networkpolicy", policy.Name))
+	_, err = virtClient.NetworkingV1().NetworkPolicies(policy.Namespace).Create(policy)
+	ExpectWithOffset(1, err).ToNot(HaveOccurred(), fmt.Sprintf("should succeed creating network policy %s/%s", policy.Namespace, policy.Name))
+}
+
+func waitForNetworkPolicyDeletion(policy *networkv1.NetworkPolicy) {
```

Comment:
Not so related to this PR, so feel free to ignore, but deleteAndWaitForNetworkPolicyDeletion, suits better this method I think.

Reply:
Well having `delete` and `Deletion` feels redundant.